cmd/operator-sdk/bundle/validate.go

```diff
@@ -132,33 +180,86 @@ func (c bundleValidateCmd) run() (err error) {
 		fields["container-tool"] = c.imageBuilder
 	}
 	logger := log.WithFields(fields)
-	if viper.GetBool(flags.VerboseOpt) {
-		log.SetLevel(log.DebugLevel)
+	if c.isToOutPutJSON() {
+		// Log as JSON instead of the default ASCII formatter.
+		log.SetFormatter(&log.JSONFormatter{})
+	} else {
+		if viper.GetBool(flags.VerboseOpt) {
+			log.SetLevel(log.DebugLevel)
+		}
 	}
-
 	val := bundle.NewImageValidator(c.imageBuilder, logger)
-
 	// Pull image if a tag was passed.
 	if c.imageTag != "" {
-		logger.Info("Unpacked image layers")
+		msg := "Unpacked image layers"
+		if c.isToOutPutJSON() {
+			jsonOutput.addLog(msg, nil)
+		} else {
+			logger.Info(msg)
+		}
 		err = val.PullBundleImage(c.imageTag, dir)
 		if err != nil {
-			logger.Fatalf("Error to unpacking image: %v", err)
+			msg := "error to unpacking image"
+			if c.isToOutPutJSON() {
+				jsonOutput.addLog(msg, err)
+				jsonOutput.printFatalJSON()
```

Comment:
It appears this idiom is used often. Would be a good idea to have a method that wraps both of these in one.

Reply:
Yep, I think we can encapsulate that.